frontend/app/(modals)/comments.tsx

```diff
@@ -1,82 +1,372 @@
 import CommentComponent from '@/components/Comment';
-import { Container, Main } from '@/tamagui.config';
-import { Feather } from '@expo/vector-icons';
+import {
```

Comment:
I see that there are a lot of imports. If there are any imports that aren't used could you remove them.
If you are on VSCode you can use the shortcut `Alt + Shift + O` to get rid of unused imports.